spec/dummy/db/schema.rb

```diff
@@ -35,6 +35,7 @@
     t.datetime "revoked_at"
     t.datetime "created_at",        null: false
     t.string   "scopes"
+    t.string   "previous_refresh_token"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.